onnxruntime/core/providers/openvino/ov_versions/data_ops.cc

```diff
@@ -0,0 +1,1047 @@
+// Copyright(C) 2019 Intel Corporation
+// Licensed under the MIT License
+
+#include <unordered_set>
+#include "core/providers/shared_library/provider_api.h"
+#include "../backend_utils.h"
+#include "../backend_manager.h"
+#include<string>
+#include<vector>
+#include "data_ops.h"
+#include "capabilities.h"
+#include "utils.h"
+
+#if defined(_MSC_VER)
+#pragma warning(disable : 4244 4245 5208)
+#elif __GNUC__
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-parameter"
+#endif
+#include <ngraph/ngraph.hpp>
+#include <ngraph/frontend/onnx_import/onnx.hpp>
+#if defined(_MSC_VER)
+#pragma warning(default : 4244 4245)
+#elif __GNUC__
+#pragma GCC diagnostic pop
+#endif
+
+namespace onnxruntime {
+namespace openvino_ep {
+
+std::set<std::string> ops_supported_only_in_model = {
+      "Cast",
+      "Concat",
+      "ConstantOfShape",
+      "Dropout",
+      "Expand",
+      "EyeLike",
+      "Exp",
+      "GatherND",
+      "Identity",
+      "NonMaxSuppression",
+      "NonZero",
+      "Not",
+      "OneHot",
+      "Pad",
+      "Range",
+      "ReduceMin",
+      "Resize",
+      "Round",
+      "Shape",
+      "Split",
+      "TopK"
+}; 
+
+std::vector<supportedop_t> supported_op_mode = {
```

Comment:
can we change the order of the elements in the struct supportedop_t.
(optype, device_type, version).

To make it more readable because we would always look for the op name first and then it's supportability

Reply:
Done